sdk/src/main/java/com/kin/ecosystem/Environment.java

```diff
@@ -75,4 +83,8 @@ public static KinEnvironment getProduction() {
 	public static KinEnvironment getPlayground() {
 		return PLAYGROUND;
 	}
+
+	public static KinEnvironment getBeta() {
+		return BETA;
```

Comment:
add documentation that Beta is for internal testing. Developers should use playground for testing and Production for release app